openqabot/types/incident.py

```diff
@@ -144,3 +148,30 @@ def contains_package(self, requires: List[str]) -> bool:
                 if package.startswith(req) and package != "kernel-livepatch-tools":
                     return True
         return False
+
+    def has_failures(self, token) -> bool:
+        results = request.get(
+            QEM_DASHBOARD + "/api/jobs/incident/" + str(self.id), headers=token
+        ).json()
+
+        # This should be wrapped by a verbose flag so we return faster
```

Comment:
I don't understand this comment. Can you elaborate?

Reply:
Well, the level of verbosity is already defined by choosing the log-level. I agree with the "info" level below so I suggest you just remove this comment then